docs/tutorial.rst

```diff
@@ -154,55 +154,91 @@ iPython Notebook before being extended to other backends.
 
 .. _Jupyter: http://jupyter.org
 
-The acbuild_ command line tool is one of the tools which can used to build
+The buildah_ command line tool is one of the tools which can used to build
 containers for rkt. It comes preinstalled by RktMachine on the CoreOS VM.
 
-.. _acbuild: https://github.com/containers/build
+.. _buildah: https://github.com/projectatomic/buildah
 
-This command uses layout in the current working directory as part of the build,
-so it is sensible to create a directory for each new container build.
+Start the container from scratch.
+
+(Note that superuser privileges are needed to run buildah commands and that the
```

Comment:
Is that on all platforms or something MacOS specific?

Reply:
No, the image building works by running a container on the current layers and executing the run lines. Running a container needs superuser or it's a trivial privilege escalation. The confusion may be because the old acbuild command was setuid.

There's an error here in the next line about that, though. The buildah command is not setuid where the old acbuild was. Figured it would be better to require explicit sudo instead. It's not too inconvenient since the .buildah.sh scripts can be run sudo.

I'll correct the documentation.